CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ if(${BLAS_ENABLE_TESTING})
 endif()
 
 option(BLAS_ENABLE_BENCHMARK "Whether to enable benchmarking" ON)
+option(BUILD_CLBLAST_BENCHMARKS "Whether to build CLBLAST benchmarks" OFF)
```

Comment:
For consistency, could this be `CLBLAST_ENABLE_BENCHMARKS` instead? Just a minor quibble.

Reply:
I didn't want it to be confusing given that `BLAS_ENABLE_BENCHMARK` acts at an higher level than `BUILD_CLBLAST_BENCHMARKS` (i.e if the value of the former is `OFF`, the latter will be ignored)